generator/mavschema.xsd

```diff
@@ -9,15 +9,14 @@
 
 
 <!-- definition of attributes -->
-<xs:attribute name="name"> <!-- enum,entry,message,field -->
+<xs:attribute name="name"> <!-- used in enum,entry,message,field elements -->
 </xs:attribute>
-<xs:attribute name="index" type="xs:unsignedByte"/> <!-- param -->
-<xs:attribute name="id" type="xs:unsignedShort"/> <!-- message -->
-<xs:attribute name="print_format" type="xs:string"/> <!-- field -->
-<xs:attribute name="enum" type="xs:string"/>
-<xs:attribute name="display" type="xs:string"/>
+<xs:attribute name="id" type="xs:unsignedShort"/> <!-- used in message elements -->
```

Comment:
Can't a message id be up to 24 bits? This is 16 bits.

Reply:
@amilcarlucas ?

Reply:
Very good point. As you can see that has been a "xs:unsignedShort" even before my changes.

So ... That is something that should be fixed by a future PR.

Reply:
Fair enough! Let's try remember to post one when this has gone in :-).

Reply:
Created https://github.com/ArduPilot/pymavlink/issues/216 for tracking